components/Caption.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+interface CaptionProps {
+  children: React.ReactNode;
+}
+
+/**
+ * @description This component displays textual captions.
+ *
+ * @component
```

Comment:
```suggestion
```

Reply:
Is `@component` coming by default in the jsdoc? If not, kindly remove it.

Reply:
Okay Sure